docs/en/Go-Client.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: global
+title: Go Client
+nickname: Go Client
+group: Clients
```

Comment:
What Gene said. When I run `jekyll serve` I see this Go doc with the other client docs in the "Clients" section